### DISCUSSION TITLE
Capsicum Support?

### DISCUSSION THREAD

**Discussion by indolering** [phylum-dev/birdcage#30] [Ideas]
Supporting more niche userland sandboxing mechanisms may or may not be a good idea but I think it would be helpful to examine additional APIs when making design choices.

Has supporting Capcisum been discussed/considered in this community yet?

**Comment by cd-work** [comment]
Since the existing APIs include sandboxing for just macOS and Linux, capsicum was not considered.
If someone wanted to add sandboxing support for BSD(s) that might one good option for APIs.

Generally one goal of Birdcage is to make sandboxing for applications simple, so an overwhelming amount of APIs to pick from each with their own advantages and drawbacks might also be detrimental. You shouldn't have to be an expert to correctly use this library, otherwise you'll just end up like pgp.
